app/controllers/organizations/invitations_controller.rb

```diff
@@ -16,6 +16,10 @@ def create
     @form = MembershipInvitation.new(email: params.dig(:membership_invitation, :email), organization: @organization, inviter: current_user)
 
     if @form.save
+      recipient = User.find_by(email: @form.email)
+      if recipient
+        MembershipInvitationNotifier.with(organization: @organization).deliver(recipient)
+      end
```

Comment:
I think it would be great to move this logic into the model. 

I think here https://github.com/yshmarov/moneygun/blob/d4d9b7e60ca609ed3720778bd6e54b17db7678b3/app/models/membership_invitation.rb#L31